data.sql

```diff
@@ -3,3 +3,20 @@
 INSERT INTO animals (name) VALUES ('Luna');
 INSERT INTO animals (name) VALUES ('Daisy');
 INSERT INTO animals (name) VALUES ('Charlie');
+
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg)VALUES ('Agumon','2020-12-03',0,true,10.23);
+
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg)VALUES ('Gabumon','2018-11-15',2,true,8);
+
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg)VALUES ('Pikachu','2021-01-07',1,false,15.04);
+
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg)VALUES ('Charmander','2020-02-08',0,false,11),
```

Comment:
- [ ] Dear @obibaadoma, are you sure you have pushed the last commit? am still wondering why this change is not been implemented yet, be aware that the number of reviews are counting down, the 1st and second reviewer mentioned that you should change `11` to `-11` as per the requirement, but in your [last commit ](https://github.com/obibaadoma/Vet_clinic_database/pull/3/commits/02807aee347fbfadd8b9a3fe260b39495c1bd1b1), nothing shows you have touched that requirement, I will still ask you to go and do it before your project get approved

Reply:
Noted. Table succsessfully updated with below script:
UPDATE animals
SET weight_kg = '-11'
WHERE name = 'Charmander';

That is the statement i used in updating it